app/controllers/search_terms_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SearchTermsController < ApplicationController
+  def index
+  end
+  
+  # GET /search_terms
+  def index
+    @search_terms = SearchTerm.all
+  end
+
+  def import
+    return redirect_to request.referer, notice: 'No file added' if params[:file].nil?
+    return redirect_to request.referer, notice: 'Only CSV files allowed' unless params[:file].content_type == 'text/csv'
+
+    CsvImportService.new.call(params[:file])
+
+    redirect_to request.referer, notice: 'Import started...'
+  end
+
+  protect_from_forgery except: :show
+  def newv
+    CsvImportService.new.call(params[:file])
+    render json: "OK"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/1d081ecd36c48dd47f0123e7164aa066b7a4d685/app/controllers/search_terms_controller.rb#L22"></span>Prefer single-quoted strings when you don't need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>